cliquet/views/errors.py

```diff
@@ -82,6 +82,10 @@ def error(context, request):
         logger.critical(context.original, exc_info=True)
         return service_unavailable(context, request)
 
+    if hasattr(request, 'raven'):
+        ident = request.raven.get_ident(request.raven.captureException())
+        print("Exception captured in event number: %s" % ident)
```

Comment:
I think it would be much better to actually configure it with: http://raven.readthedocs.org/en/latest/integrations/logging.html or http://raven.readthedocs.org/en/latest/integrations/pyramid.html

I will rework that.